tests/security/selinux/selinuxexeccon.pm

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test '# selinuxexeccon' command works,
+#          report process context for command from the current context
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#105202, tc#1769801
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+
```

Comment:
how about removing the blank line?

Reply:
I added this blank line on purpose. 
The "my .." part is definition module,
then followed by a "blank line"
then follwed by "real code".
It is readable.